test/framework/utils/utils.go

```diff
@@ -25,3 +25,11 @@ func NamespacedName(obj metav1.Object) types.NamespacedName {
 		Name:      obj.GetName(),
 	}
 }
+
+func CopyMap(original map[string]string) map[string]string {
```

Comment:
TODO for later - These `utils` _usually_ could be shared across the project instead of one in tests and src.
In this case, I notice these aren't test specific too.

Reply:
Moved it to utils.